recipes/chembl_webresource_client/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "chembl_webresource_client" %}
+{% set version = "0.9.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/5d/43/bf0a4d475be9b16f1cbecf0dfe5cb2652305ad239d726a9238d1c4d1713d/chembl-webresource-client-0.9.31.tar.gz
```

One small improvement here. Could you please change it to:
```suggestion
  url: https://files.pythonhosted.org/packages/5d/43/bf0a4d475be9b16f1cbecf0dfe5cb2652305ad239d726a9238d1c4d1713d/chembl-webresource-client-{{ version }}.tar.gz
```